src/main/kotlin/org/jetbrains/research/testspark/tools/TestBodyPrinterFactory.kt

```diff
@@ -0,0 +1,17 @@
+package org.jetbrains.research.testspark.tools
+
+import org.jetbrains.research.testspark.core.test.SupportedLanguage
+import org.jetbrains.research.testspark.core.test.TestBodyPrinter
+import org.jetbrains.research.testspark.core.test.java.JavaTestBodyPrinter
+import org.jetbrains.research.testspark.core.test.kotlin.KotlinTestBodyPrinter
+
+class TestBodyPrinterFactory {
+    companion object {
+        fun createTestBodyPrinter(language: SupportedLanguage): TestBodyPrinter {
+            return when (language) {
+                SupportedLanguage.Kotlin -> KotlinTestBodyPrinter()
+                SupportedLanguage.Java -> JavaTestBodyPrinter()
+            }
+        }
+    }
+}
```

Comment:
**[Thinking out load here; not for this PR but worth investing effort at]:**

Every `[Something]Factory` class that we have often contains a single method `create[Something](language)`. We can turn the **Factory Pattern** into the **Factory Method Pattern**.

These `create[Something]()` static methods can be moved into the base abstract classes/interfaces (e.g., in this case we can place `createTestBodyPrinter()` static method into `TestBodyPrinter` interface (and maybe rename to just `create()`). This will allow us to remove a single entity from the codebase, namely `TestBodyPrinterFactory`. It'll reduce the code complexity.

Reply:
Addressed in https://github.com/JetBrains-Research/TestSpark/issues/305